### PR TITLE
Feat: Update Mqtt topic endpoints

### PR DESCRIPTION
**Pull request by msarcev**
# Description of change

- Updated Mqtt topic endpoints Tip-28 spec

OpenApi [here](https://studio.asyncapi.com/?url=https://raw.githubusercontent.com/iotaledger/tips/stardust-event-api/tips/TIP-0028/event-api.yml#operation-subscribe-messages)
Reference impl in go [here](https://github.com/gohornet/inx-mqtt/blob/develop/topics.go#L28)

Aims to solve task https://github.com/iotaledger/iota.js/issues/827

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

N/A

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: packages/mqtt/src/clients/mqttClient.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ export class MqttClient implements IMqttClient {
      * @returns A subscription Id which can be used to unsubscribe.
      */
     public milestonesLatest(callback: (topic: string, data: IMqttMilestoneResponse) => void): string {
-        return this.internalSubscribe("milestones/latest", true, callback);
+        return this.internalSubscribe("milestone-info/latest", true, callback);
```
Comment (lzpap) [inline packages/mqtt/src/clients/mqttClient.ts:106]: please add `milestoneId` to `IMqttMilestoneResponse`

**Review by laumair — APPROVED** [review]

File: packages/mqtt/src/clients/mqttClient.ts [hunk 2]
```diff
@@ -212,7 +212,7 @@ export class MqttClient implements IMqttClient {
      * @returns A subscription Id which can be used to unsubscribe.
      */
     public messagesMilestoneRaw(callback: (topic: string, data: Uint8Array) => void): string {
-        return this.internalSubscribe<Uint8Array>("messages/milestone", false, callback);
```
Comment (lzpap) [inline packages/mqtt/src/clients/mqttClient.ts:215]: this return binary milestone payload, not a message

File: packages/mqtt/src/clients/mqttClient.ts [hunk 3]
```diff
@@ -221,7 +221,7 @@ export class MqttClient implements IMqttClient {
      * @returns A subscription Id which can be used to unsubscribe.
      */
     public messagesMilestone(callback: (topic: string, data: IMessage) => void): string {
-        return this.internalSubscribe<Uint8Array>("messages/milestone", false, (topic, raw) => {
+        return this.internalSubscribe<Uint8Array>("milestones", false, (topic, raw) => {
             callback(topic, deserializeMessage(new ReadStream(raw)));
```
Comment (lzpap) [inline packages/mqtt/src/clients/mqttClient.ts]: Returns milestone payload in serialized form, not a message

**Review by lzpap — APPROVED** [review]

**Comment by msarcev** [comment]
Also changes `packages/iota` for additional exports needed.
`@iota/iota` should be published first and then new version updated in `packages/mqtt/package.json` ?

**Pull request merged**